MultiServer.py

```diff
@@ -222,11 +222,11 @@ def _init_game_data(self):
             self.all_item_and_group_names[game_name] = \
                 set(game_package["item_name_to_id"]) | set(self.item_name_groups[game_name])
 
-    def item_names_for_game(self, game: str) -> typing.Dict[str, int]:
-        return self.gamespackage[game]["item_name_to_id"]
+    def item_names_for_game(self, game: str) -> typing.Optional[typing.Dict[str, int]]:
+        return self.gamespackage[game]["item_name_to_id"] if game in self.gamespackage else None
```

Comment:
might empty dict be a better return here?

Reply:
you couldn't differentiate between unknown game ("Hint for ID" message) and unknown item ("No item/location found" message) then. The code below would have to `if game in self.gamespackage` rather than `if item_names_for_game(game) is not None`, at which point we can also leave `item_names_for_game` as is.